memegen/routes/_common.py

```diff
@@ -22,13 +22,11 @@ def samples():
     """Generate dictionaries of sample image data for template rendering."""
     for template in sorted(current_app.template_service.all()):
         path = template.sample_path
-        url = route('image.get', key=template.key, path=path, _external=True)
-        link = route('links.get', key=template.key, path=path)
```

Comment:
Yes that would work fine. The link would display as:

```
http://memegen.link/afraid/i-don't-know-what-this-meme-is-for/and-at-this-point-i'm-too-afraid-to-ask
```

instead of how it currently is:

```
/afraid/i-don't-know-what-this-meme-is-for/and-at-this-point-i'm-too-afraid-to-ask
```

I can also make the link intelligently use http or https if you desire the former link format:

```python
link = route('links.get', key=template.key, path=path, _external=True,
             _scheme=request.scheme)
```